builder/tests/builder/src/main/java/io/helidon/builder/test/testsubjects/Level2Blueprint.java

```diff
@@ -29,36 +29,36 @@
 interface Level2Blueprint extends Level1Blueprint {
 
     /**
-     * Used for testing and demonstrating {@link io.helidon.config.metadata.ConfiguredOption} settings.
+     * Used for testing and demonstrating option settings.
      *
      * @return ignored, here for testing purposes only
      */
     @Override
-    @ConfiguredOption("2")
+    @Option.Default("2")
     String getLevel0StringAttribute();
 
     /**
-     * Used for testing and demonstrating the use of {@link io.helidon.builder.api.Prototype.Singular}.
+     * Used for testing and demonstrating the use of {@link io.helidon.builder.api.Option.Singular}.
      *
      * @return ignored, here for testing purposes only
      */
-    @Prototype.Singular
+    @Option.Singular
```

Comment:
Option seems appropriate for values, but I feel `Prototype` is/was more apropos for `Singular` as it aligns to the generated code.

Reply:
But the option is singular, not the prototype

Reply:
Please check the usage - it looks logical, that all option (method) related annotations are prefixed with `Option`, and all prototype (class) related annotations are prefixed with `Prototype`.